cpp/ql/test/library-tests/ir/ir/aliased_ir.ql

```diff
@@ -0,0 +1,11 @@
+/**
+ * @kind graph
+ */
+
+private import cpp
+private import semmle.code.cpp.ir.implementation.aliased_ssa.PrintIR
+private import PrintConfig
+
+private class PrintConfig extends PrintIRConfiguration {
+  override predicate shouldPrintDeclaration(Declaration decl) { shouldDumpDeclaration(decl) }
+}
```

Comment:
It would have probably been better to have this commit _before_ b44c4587a40d121b31d9ddfaecdd32a497e846b9 so that the difference in test output was visible in 404145dd1b36e1668aef976b4e4e04553f8cbfc9 😭. Nevertheless, I think I've been able to convince myself that the changes to the IR look okay 👍.